snakefood3/utils.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from typing import Generator, Union
+
+
+class Utils:
+    @classmethod
```

Comment:
what's the point adding a extra level namespace?

Reply:
moved it to same class